.github/workflows/scala.yml

```diff
@@ -1,17 +1,36 @@
-name: CI
+name: Scala CI
 
 on: [push, pull_request]
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
-
+    services:
+      postgres:
+        image: postgres:9.6
+        ports: ["5432:5432"]
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
     steps:
-    - uses: actions/checkout@v1
-    - name: Run a one-line script
-      run: echo Hello, world!
-    - name: Run a multi-line script
-      run: |
-        echo Add other actions to build,
-        echo test, and deploy your project.
+      - uses: actions/checkout@v1
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - uses: olafurpg/setup-scala@v2
+      - name: Install PostgreSQL 9.6 client
+        run: |
+          sudo apt update
+          sudo bash -c "echo deb http://apt.postgresql.org/pub/repos/apt/ bionic-pgdg main >> /etc/apt/sources.list.d/pgdg.list"
+          wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+          sudo apt-get update
+          sudo apt-get -yqq install libpq-dev postgresql-client-9.6
+      - name: Setup db
+        run: |
+          psql -c 'create database travis_ci_test;' -U postgres -h 127.0.0.1
```

Comment:
Maybe we could rename `travis_ci_test`.  I was confused why travis is involved at first.

Reply:
I don't think we even need `travis_ci_test` anymore and this can probably be removed? It was something I was experimenting with when trying to set up travis.